cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -86,7 +88,7 @@ public class EntityTopicOperator extends AbstractModel implements SupportsLoggin
     private String featureGatesEnvVarValue;
 
     private String watchedNamespace;
-    /* test */ long reconciliationIntervalMs;
+    /* test */ Optional<Long> reconciliationIntervalMs;
```

Comment:
I would probably just use `null` to indicate that it is not set instead of using `Optional<Long>`. That is how this kind of things is handled in the code base in most situations I think.